content/references/protocol-reference/transactions/transaction-types/nftokenmint.md

```diff
@@ -49,7 +48,6 @@ This transaction assumes that the issuer, `rNCFjv8Ek5oDrNiMJ3pw6eLLFtMjZLJnf2`,
 | Field         | JSON Type           | [Internal Type][] | Description        |
 |:--------------|:--------------------|:------------------|:-------------------|
 | `NFTokenTaxon` | Number | UInt32 | The taxon associated with the token. The taxon is generally a value chosen by the minter of the token. A given taxon can be used for multiple tokens. Taxon identifiers greater than `0xFFFF'FFFF` are disallowed. |
-| `Issuer` | String | AccountID | _(Optional)_ The issuer of the token, if the sender of the account is issuing it on behalf of another account. This field must be omitted if the account sending the transaction is the issuer of the `NFToken`. If provided, the issuer's [AccountRoot object][] must have the `NFTokenMinter` field set to the sender of this transaction (this transaction's `Account` field). |
```

Comment:
I thought this field was necessary when acting as a minter.

Reply:
Yes but per the Git issue, it's a complex example to start with, given that there's an example of an authorized minter later in the topic.

Reply:
I get removing it from the example above but this block is what lists all possible fields.  Without this block there is no full definition of the field.